src/program.ts

```diff
@@ -336,6 +336,8 @@ export class Program extends DiagnosticEmitter {
   typeClasses: Map<TypeKind,Class> = new Map();
   /** Managed classes contained in the program, by id. */
   managedClasses: Map<i32,Class> = new Map();
+  signatureTypes: Signature[] = new Array<Signature>(0);
+  signatureID: i32 = 0;
```

Comment:
Perhaps `nextSignatureId` and move it right below `nextClassId` for consistency.

Reply:
Okay. Easy enough.

Reply:
All set.